index.md

```diff
@@ -59,6 +59,11 @@ JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
 and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
 
+## <a name="presentations"></a>Presentations
+Here is a selection of Janusgraph presentations:
+* <a href="https://www.slideshare.net/ptgoetz/large-scale-graph-analytics-with-janusgraph">DataWorksJun2017: Large Scale Graph Analytics with JanusGraph</a>, P. Taylor Goetz, 2017.06.13
```

Comment:
You may need a blank line before a bulleted list for it to render properly; do you have a preview link for your change? See the top-level README for instructions.